### PR TITLE
Added docs for hero, header component

### PR DESCRIPTION
**Pull request by gsalunke**
@Arpita-Jaiswal plz review this.

I have started adding documentation for components developed for admint.io

Task:
https://app.asana.com/0/1204213004828720/1204270246677219/f

### CODE REVIEW

File: FASTN.ftd [hunk 1]
```diff
@@ -21,3 +23,10 @@
 # Pricing🚧: 
 
 # Learn: learn/
+
+# Docs: docs/
+skip: true
+
+- Docs: docs/
+- Hero: docs/hero/
+- Header: docs/header/
```
Comment (Arpita-Jaiswal) [inline FASTN.ftd:32]: Add a new line

File: docs/index.ftd [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+This document contains various components we have developed using `ftd`
+language.
+
+Below are few sample components.
+
+-- ds.h3: Header Component
+
+We have developed header component of admint.io as per design present currently
+on [admint.io](https://admint.io/)live site.
```
Comment (gargajit) [inline docs/index.ftd:35]:
```suggestion
on [admint.io](https://admint.io/) live site.
```

**Review by gargajit — CHANGES_REQUESTED** [review]

File: docs/index.ftd [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+-- ds.page.body:
+
+This document contains various components we have developed using `ftd`
+language.
+
+Below are few sample components.
+
+-- ds.h3: Header Component
+
+We have developed header component of admint.io as per design present currently
```
Comment (gargajit) [inline docs/index.ftd:34]:
```suggestion
We have developed the `header` component of admint.io as per the design present currently
```

**Review by gargajit — CHANGES_REQUESTED** [review]

**Review by Arpita-Jaiswal — CHANGES_REQUESTED** [review]

File: docs/index.ftd [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+-- footer:
+
+-- end: ds.page.footer
+
+-- ds.page.body:
+
+This document contains various components we have developed using `ftd`
+language.
+
+Below are few sample components.
```
Comment (gargajit) [inline docs/index.ftd:30]:
```suggestion
Below are a few sample components:
```

**Review by gargajit — CHANGES_REQUESTED** [review]

File: docs/index.ftd [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+sections: $lib.sitemap.sections
+
+-- end: ds.rendered.output
+
+-- end: ds.rendered
+
+-- ds.h3: Hero Component
+
+We have developed hero component of admint.io as per design present currently
+on [admint.io](https://admint.io/) live site.
```
Comment (gargajit) [inline docs/index.ftd:64]:
```suggestion
We have developed the `hero` component for [admint.io](https://admint.io/), which matches the design currently displayed on the live site.
```

File: docs/header.ftd [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+as give below:
+
+-- ds.code:
+lang: ftd
+
+\-- import: fifthtry.github.io/admint.io/lib
+
+-- ds.markdown:
+
+Once you import `lib` inside your `.ftd` file, use `header` component by adding
```
Comment (gargajit) [inline docs/header.ftd:51]:
```suggestion
Once you import `lib` inside your `.ftd` file, use the `header` component by adding
```

File: docs/hero.ftd [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+-- docs.footer:
+
+-- end: ds.page.footer
+
+-- ds.page.body:
+
+[See it on full page](index.html)
+
+We have developed hero component of admint.io as per design present currently
+on [admint.io](https://admint.io/) live site.
```
Comment (gargajit) [inline docs/hero.ftd:31]:
```suggestion
We have developed the `hero` component for [admint.io](https://admint.io/), 
which matches the design currently displayed on the live site.
```

File: docs/hero.ftd [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+
+-- ds.h3: How to use?
+
+To use hero component you need to `import` `lib.ftd` inside your `.ftd` file, as
```
Comment (gargajit) [inline docs/hero.ftd:35]:
```suggestion
To use the `hero` component you need to import `lib.ftd` inside your `.ftd` file, as
```

**Review by gargajit — APPROVED** [review]

File: docs/index.ftd [hunk 9]
```diff
@@ -0,0 +1,114 @@
...
+
+-- fastn-footer.fastn-footer:
+copyright: Copyright © 2023 - [FifthTry](https://www.fifthtry.com/)
+site-logo: $fastn-footer.fastn-logo.light
+powered-by-text: Powered by
+powered-by-link: https://fastn.io/
+
+-- end: ftd.column
+
+-- end: footer
```
Comment (Arpita-Jaiswal) [inline docs/index.ftd:114]:
```suggestion
-- end: footer

```

Add a new line here

**Review by Arpita-Jaiswal — CHANGES_REQUESTED** [review]

**Review by Arpita-Jaiswal — APPROVED** [review]

**Pull request merged**